cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,57 @@
+
+Cockroach Labs cannot provide specific advice about what is required for compliance with PCI DSS or how to implement a specific requirement. The following points help to illustrate some steps that organizations might take to help to ensure compliance..
+
+{{site.data.alerts.callout_danger}}
+The following list is provided only to illustrate a small subset of the measures you might need to take to ensure compliance.
+{{site.data.alerts.end}}
+
+- Before you insert card data into the cluster, you must first protect it by a combination of encryption, hashing, masking, and truncation. For an example implementation, refer to [Integrate CockroachDB Dedicated with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+- The cryptographic materials used to protect card data must themselves be protected at rest and in transit, and access to the unencrypted key materials must be strictly limited only to approved individuals.
+- Within the cluster, you must restrict access to card data on a "need to know basis" authorized individuals have access to tables and views in the cluster that contain card data, and you must regularly test for compliance. Refer to [Authorization](/docs/{{site.versions["stable"]}}/authorization.html).
+- You must track, monitor, and log all access to card data and network resources. Within a cluster with PCI DSS enabled, {{ site.data.products.db }} enforces this requirement automatically. Refer to [Security and audit monitoring](/docs/{{site.versions["stable"]}}/logging-use-cases.html#security-and-audit-monitoring).
+- You must protect networks that transmit card data from malicious access over the public internet, and you must regularly test for compliance. For more information about protecting the cluster's networks, refer to [Network Authorization](network-authorization.html).
+- You must regularly test all systems, applications, and dependencies (including, but not limited to, application libraries, runtimes, container images, and testing frameworks) that are involved in storing or processing card data for known vulnerabilities, you must regularly apply updates, and you must regularly test for compliance. Important security and stability updates are applied regularly and automatically to {{ site.data.products.dedicated }} clusters. These updates include, but are not limited to, the cluster's CockroachDB runtime, the operating systems of cluster nodes, APIs, and management utilities. You are notified about upcoming cluster maintenance before it happens, when it starts, and when it completes.
```

Comment:
Notified how? An email list... callouts on the website or docs...
Should we link here to release notes and/or technical advisories? https://www.cockroachlabs.com/docs/advisories/index.html

Reply:
They are notified by whatever mechanisms they set up in Cockroach Cloud, I think. Let me see what I can do.